### ISSUE TITLE
No! Don't do this now! (Installation path)

### ISSUE THREAD

**Issue by GasmoN** [GetPublii/Publii#724]
I know that I am not the only one so please don't install publii by yourself. Let me choose installation directory. I have seperated hard disks for windows, applications and games, I have everything in order and organized I don't know what's the point of automatically installing software on user's system instead of give them choice where to install it.

**Comment by renanlecaro** [comment]
Have you tried setting the storage folder in the app ? 
https://getpublii.com/docs/publii-global-settings.html

**Comment by GasmoN** [comment]
I am talking about publii's installation path. When I click on installer, it's automatically installs Publii under AppData folder instead to give me choice to choose installation path.

**Comment by renanlecaro** [comment]
Oh alright, I'm on Linux so we just run the executable directly without
installation there. Please note that I'm just one of the users, not a
maintainer.

**Comment by nikahmadz** [comment]
I agree. can't proceed with installation. I have set my anti-ransom-ware software to block third-party access to `C:\Users\USER NAME\Documents\` folder. [Publii failed to access this folder](https://getpublii.com/docs/install-publii.html).
